.github/workflows/production-suite.yml

```diff
@@ -2,7 +2,21 @@ name: Zulip Production Suite
 
 on:
   push:
+    paths:
+      - "**/migrations/**"
+      - puppet/**
+      - scripts/**
+      - tools/**
```

Comment:
Other paths that belong here:
* requirements
* yarn.lock
* static/ (Building static assets with the production config is only tested in this suite; so maybe we could exclude that later since it's going to lead to a lot of rebuilds)
* zproject/